src/angularjs/src/app/analysis-jobs/list/analysis-jobs-list.controller.js

```diff
@@ -41,10 +41,10 @@
         }
 
         function filterJobs(filters) {
-            var params = _.merge({}, defaultParams, filters);
```

Comment:
Filters was only ever set via the watcher, it was removed since we manually set both properties we get from the filters object below